pyseir/rt/patches.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+import numpy as np
+
+from libs.datasets.combined_datasets import load_us_latest_dataset
+import pyseir.utils
+
+population_dict = load_us_latest_dataset().data.set_index("fips")["population"].to_dict()
```

Comment:
I'd prefer not to load this here - in general I like to keep imports real lightweight.  when you're reading from disk during a file import, I find it make it more difficult to test + slows down initialization. 

couple options:
 - just run this code in the `patch aggregate rt results` function.  I don't think it should be too expensive.
 - Just call `population = combined_datasets.get_us_latest_for_fips(fips)[CommonFields.POPULATION]` inside of load_and_append_population - should also be speedy

Reply:
Fair point -> implemented the second one.